ldms/src/decomp/as_is/decomp_as_is.c

```diff
@@ -256,20 +256,26 @@ __decomp_as_is_config(ldmsd_strgp_t strgp, json_entity_t jcfg,
 	TAILQ_FOREACH(jidx, &jidxs->item_list, item_entry) {
 		didx = &dcfg->idxs[j];
 		if (jidx->type != JSON_DICT_VALUE) {
-			DECOMP_ERR(reqc, EINVAL, "an index must be an object\n");
+			DECOMP_ERR(reqc, EINVAL, "strgp '%s': index '%d': "
```

Comment:
I think something more like "expected the index list entries to be dictionaries." I don't know that users know what an "object" is in the context of JSON.